### PR TITLE
✨ Lentainform. this change is very important

### PR DESCRIPTION
**Pull request by DenyHF**
Hello.
These changes have been lost in previous commits.

### CODE REVIEW

**Comment by DenyHF** [comment]
@zhouyx 
Hello
Can you approve this request? thank you.

**Comment by zhouyx** [comment]
Hi @DenyHF Sorry for the delay in review.  I am currently traveling. 
To @lannka and the ads team.

**Comment by DenyHF** [comment]
@zhouyx, thank you for fast answer.
Have a nice traveling.

File: ads/lentainform.js [hunk 1]
```diff
@@ -35,5 +35,12 @@ export function lentainform(global, data) {
     `${encodeURIComponent(data.widget)}.js?t=` +
     Math.floor(Date.now() / 36e5);
 
+  window.context.observeIntersection(function(changes) {
```
Comment (lannka) [inline ads/lentainform.js]: use `global` please

Comment (DenyHF) [reply ads/lentainform.js]: done

**Pull request merged**